app/forms/idv/profile_form.rb

```diff
@@ -35,11 +38,11 @@ def profile
     end
 
     def pii_attributes
-      @_pii_attributes ||= Pii::Attributes.new
+      @_pii_attributes ||= Pii::Attributes.new_from_hash(previous_address: OpenStruct.new)
```

Comment:
I would agree with you if the difference was negligible, but a 50% increase in speed is pretty significant for a small loss in readability. Note that this is not our first use of this pattern in this app. See https://github.com/18F/identity-idp/blob/master/app/forms/idv/finance_form.rb#L79